test/spec/index.js

```diff
@@ -58,10 +58,15 @@ describe('Clay', function() {
     function() {
       stubPebble();
       var clay = fixture.clay([]);
+
+      // we stub the generateUrl method to avoid very large string comparisons.
+      var generateUrlStub = sinon.stub(clay, 'generateUrl');
+      generateUrlStub.returns('data:text/html;base64,PGh0bWw%2BVEVTVDwvaHRtbD4%3D');
       Pebble.addEventListener.withArgs('showConfiguration').callArg(1);
 
       assert(Pebble.addEventListener.calledWith('showConfiguration'));
       assert(Pebble.openURL.calledWith(clay.generateUrl()));
+      generateUrlStub.restore();
```

Comment:
In this circumstance there are no side effects because `fixture.clay([]);` returns a new instance. I'm happy to leave it the way it is. I'll remember to do stubbing etc outside the test for things that do get recycled between tests.